renku_notebooks/api/schemas/servers_get.py

```diff
@@ -361,6 +367,52 @@ def get_status(server: UserServerManifest):
                     if step["status"] in [StepStatusEnum.ready.value]
                 ]
             )
+
+            output["warnings"] = []
+
+            if server.using_default_image:
+                output["warnings"].append(
+                    {"message": "Server was started using the default image."}
+                )
+
+            idle_seconds = int(server.manifest.get("status", {}).get("idleSeconds", 0))
+            idle_threshold = (
+                server.manifest.get("spec", {})
+                .get("culling", {})
+                .get("idleSecondsTheshold", 0)
+            )
+            remaining_idle_time = idle_threshold - idle_seconds
+
+            if idle_threshold > 0 and remaining_idle_time < 60 * 60:
```

Comment:
I think here you want the values we set in the config, right? You have 1 hour hardcoded here.